Service/CreateMarriageService.php

```diff
@@ -59,22 +65,28 @@ class CreateMarriageService
 
     /**
      * @param EntityManagerInterface $entityManager          The Entity Manager
+     * @param CacheService           $cacheService           The Cache Service
      * @param HandleAssentService    $handleAssentService    The Handle Assent Service
      * @param UpdateChecklistService $updateChecklistService The Update Checklist Service
      * @param Security               $security               The Security
+     * @param LoggerInterface        $logger                 The Logger Interface
      */
     public function __construct(
         EntityManagerInterface $entityManager,
+        CacheService $cacheService,
         HandleAssentService $handleAssentService,
         UpdateChecklistService $updateChecklistService,
-        Security $security
+        Security $security,
+        LoggerInterface $logger
```

Comment:
If you rename $logger here to $pluginLogger logs are written nicely to the plugin-channel.